test/parallel/test-http-client-abort-no-agent.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+const net = require('net');
+
+const server = http.createServer(() => {
+  throw new Error('test invalidation');
```

Comment:
Can you use `common.fail()` instead of explicitly throwing (in the other test too).